### PR TITLE
Replace fireTestRunStarted/Finished by fireTestSuiteStarted/Finished in XtFileRunner

### PR DESCRIPTION
**Pull request by jean-andre-gauthier**
This PR replaces calls to `fireTestRunStarted/Finished` by `fireTestSuiteStarted/Finished` in `XtFileRunner`. According to the [JUnit 4 docs](https://github.com/junit-team/junit4/blob/16228f3ccea3c6f1170488e0e268f3601d130f75/src/main/java/org/junit/runner/notification/RunNotifier.java#L87), `fireTestRunStarted/Finished` are not supposed to be invoked by custom runners, which should use `fireTestSuiteStarted/Finished` instead.

### CODE REVIEW

**Comment by jean-andre-gauthier** [comment]
@marcphilipp could you please double-check if the changes correspond to what you had in mind?

File: tests/org.eclipse.n4js.ide.tests.helper.tests/src/org/eclipse/n4js/ide/tests/helper/server/xt/tests/BeforeAfterInvocationTest.java [hunk 1]
```diff
@@ -37,25 +37,27 @@ public class BeforeAfterInvocationTest extends AbstractXtParentRunnerTest {
...
+
 	@Test
 	public void test() throws Exception {
 		assertNull(invocationRecorder);
 		invocationRecorder = new ArrayList<>();
 
 		run(new BeforeAfterTestRunSimulator(), "probands/BeforeAfterInvocation");
-		assertEventNames("testRunStarted\n"
+		assertEventNames("testSuiteStarted\n"
+				+ "testSuiteStarted\n"
```
Comment (marcphilipp) [inline tests/org.eclipse.n4js.ide.tests.helper.tests/src/org/eclipse/n4js/ide/tests/helper/server/xt/tests/BeforeAfterInvocationTest.java:48]: ❓ Why is this emitted twice now and was only once before?

**Comment by mmews-n4** [comment]
@jean-andre-gauthier @marcphilipp Your initiative and help is appreciated. Thanks! Regarding the test output: I also was wondering why the test suite was started twice. When you un-draft this PR, I will have a look at it right away.

**Review by marcphilipp — APPROVED** [review]

**Comment by mmews-n4** [comment]
Thanks again!

**Pull request merged**